common/changes/office-ui-fabric-react/unknownPersonaCoin_2018-05-08-13-17.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added support for special casing UnknownPersona coin",
+      "type": "minor"
```

Comment:
Its backwards compatible, so can be a patch

Reply:
But it has a (non-breaking) API change, which qualifies it as minor according to https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/Contributing/ChangeFiles.md
>Any API change should qualify as at least as a minor change. When you add a new API in a backward compatible manner. i.e. Use of the new API is optional.

Reply:
I stand corrected!